cinema/views.py

```diff
@@ -21,22 +24,39 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+):
```

Comment:
The `GenreViewSet` should use the `AllowListAndCreateOnly` permission to restrict actions to list and create, as specified in the task requirements.